doc/guide/bindings.hpp

```diff
@@ -425,8 +493,10 @@ Input C++ (full program, 'random_numbers_main.cpp'):
       "The output random numbers can be saved with the " +
       PRINT_PARAM_STRING("output") + " output parameter.  In addition, a "
       "randomly generated linear regression model can be saved with the " +
-      PRINT_PARAM_STRING("output_model") + " output parameter."
-      "\n\n"
+      PRINT_PARAM_STRING("output_model") + " output parameter.");
+
+// Example.
+BINDING_EXAMPLE(
```

Comment:
```suggestion
  // Example.
  BINDING_EXAMPLE(
```
A similar spacing fix as above. :+1: